packages/react/src/constants/device-sizes.constant.ts

```diff
@@ -71,7 +87,7 @@ export const getSizesForBreakpoints = ({ small, medium }: Breakpoints) => {
 
   const largeMin = newSizes.medium.max + 1;
   newSizes.large = {
-    max: 2000, // TODO: decide upper limit
+    max: largeMin + 1,
```

Comment:
I would undo the change here to avoid any surprises.

Reply:
done